tests/Doctrine/Tests/DBAL/Query/QueryBuilderTest.php

```diff
@@ -87,14 +87,25 @@ public function testSelectWithLeftJoin() : void
         self::assertEquals('SELECT u.*, p.* FROM users u LEFT JOIN phones p ON p.user_id = u.id', (string) $qb);
     }
 
+    public function testSelectWithLeftJoinNoCondition() : void
```

Comment:
One test is enough. If something goes wrong, there's no point in seeing three tests failing.

Reply:
Done!